packages/visx-brush/src/BrushHandle.tsx

```diff
@@ -12,12 +12,23 @@ export type BrushHandleProps = {
   onBrushEnd?: (brush: BrushState) => void;
   type: ResizeTriggerAreas;
   handle: { x: number; y: number; width: number; height: number };
+  isUseWindowMoveEvents?: boolean;
```

Comment:
Same suggestion here 
```suggestion
  useWindowMoveEvents?: boolean;
```